libraries/AP_RCProtocol/examples/RCProtocolTest/RCProtocolTest.cpp

```diff
@@ -535,9 +535,6 @@ void loop()
     printf("Test count %u - %u failures\n", unsigned(test_count), unsigned(test_failures));
 }
 
-const struct AP_Param::GroupInfo        GCS_MAVLINK_Parameters::var_info[] = {
```

Comment:
i notice a lot of changes to these example targets, have you tested any of them?

Reply:
Yes, in SITL.